extensions/eventtap/event.m

```diff
@@ -406,6 +406,60 @@ static int eventtap_event_newKeyEvent(lua_State* L) {
     return 1;
 }
 
+/// hs.eventtap.event.newScrollWheelEvent(offsets, mods, unit) -> event
+/// Constructor
+/// Creates a scroll wheel event
+///
+/// Parameters:
+///  * offsets - A table containing the {vertical, horizontal} amount to scroll
```

Comment:
Even though vertical scrolling is much more common, I think it'd be better to follow the convention of having the offsets be {horizontal,vertical}.
I also think it would be useful to indicate which directions the positive/negative variants of each offset, scroll in.